sqlmesh/core/config/loader.py

```diff
@@ -6,6 +6,7 @@
 from pathlib import Path
 
 from pydantic import ValidationError
+from dotenv import load_dotenv
```

Comment:
This is a new dependency that needs to be included in `pyproject.toml`, right?

Reply:
Yes, let's not rely on transitivity